demo-shell/src/app/components/card-view/card-view.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="app-main-content">
     <h1>CardView Component</h1>
 
-    <mat-card class="app-card-view">
+    <mat-card appearance="outlined" class="app-card-view">
```

Comment:
this one will be removed before migration by https://github.com/Alfresco/alfresco-ng2-components/pull/9050